### PR TITLE
UI-updates for summer 2022

### PR DESCRIPTION
**Pull request by mhieta**
## Description

This PR will add the requested UI-updates for the beginning of summer 2022.

## Context

These changes will give better application wide notifications support (with localization) and it provides links for retrieving more detailed water quality information regarding beaches.

## Screenshots

Aurinkolahden uimaranta | Ulkoliikuntakartta 2022

### CODE REVIEW

File: src/domain/i18n/locales/sv.json [hunk 1]
```diff
@@ -75,11 +75,12 @@
     "TEMPERATURE": "Temperatur",
     "LINKS": "Länkar",
     "GET_ROUTE": "Väg hit",
+    "EXTRA_INFO": "Se information om badvattenkvalitet",
     "NOTICE": "Meddelande",
     "WATER_TEMPERATURE": "Vattnets temperatur",
     "WATER_TEMPERATURE_SENSOR": "Vattentemperatur sensor",
     "CLOSE": "Stänga",
-    "SEE_ON_SERVICE_MAP": "Se på Palvelukartta"
+    "SEE_ON_SERVICE_MAP": "Mer information om servicekarta"
```
Comment (nikomakela) [inline src/domain/i18n/locales/sv.json:83]: I would say that this translation is a bad one: This is like "Get more information about servicemap". I'm not a Swedish speaking guy, but I would say "Mer information på servicekartan" would be better.

```suggestion
    "SEE_ON_SERVICE_MAP": "Mer information på Servicekartan"
```

**Review by nikomakela — COMMENTED** [review]
I think it would be good after a Swedish text change that I suggested.

File: src/domain/i18n/locales/sv.json (hunk 1 above)
Comment (mhieta) [reply src/domain/i18n/locales/sv.json]: @nikomakela Good point, swedish translation updated now

**Review by nikomakela — APPROVED** [review]
Good job

**Pull request merged**